plugins/generator-1.19.2/forge-1.19.2/templates/json/tool.json.ftl

```diff
@@ -7,4 +7,14 @@
             "layer0": "${modid}:items/${data.texture}"
         </#if>
     }
+<#if data.getModElement().getTypeString() == "tool" && data.toolType == "Shield">,

```

Comment:
Indentations here and probably in other files still need some work

Reply:
This and matching closing operators should start with 4 spaces, all inside them should be shifted by 6 spaces I guess.

Reply:
I'd love to see a visual example 😅